examples/web-wagmi/src/misc/LoginSpecificProfile.tsx

```diff
@@ -0,0 +1,31 @@
+import { useActiveProfile } from '@lens-protocol/react';
+import { useState } from 'react';
+import { Navigate } from 'react-router-dom';
+
+import { LoginButton } from '../components/auth/LoginButton';
+
+export function LoginSpecificProfile() {
+  const [handle, setHandle] = useState('');
+  const activeProfile = useActiveProfile();
+
+  return (
+    <>
+      {activeProfile.data && <Navigate to="/" />}
+
+      <h2>Login to a Specific Profile</h2>
+
+      <p>Handle</p>
```

Comment:
I guess better to use `useProfilesOwnedBy` hook here and display a list of owned profiles instead of just an input.

Reply:
Is there a a way to get the wallet address without triggering the sign-in flow? (Not sure, generally asking for a way to do so)